modules/react/tabs/stories/Tabs.stories.mdx

```diff
@@ -108,84 +106,149 @@ the end.
 
 ### Tabs
 
-The `Tabs` component is the container component that is responsible for creating a `TabsModel` that
-is shared using React context. It does not represent a real element.
-
 #### Usage
 
+`Tabs` is a container component that is responsible for creating a `TabsModel` and sharing it with
+its children using React context. It does not represent a real element.
+
 ```tsx
 // configuration
-<Tabs onActivate={({ data }) => console.log('Activate tab', data.tab)}>
-  {/* sub components go here */ }
-</Tabs>
+<Tabs onActivate={({data}) => console.log('Activate tab', data.tab)}>
+  {/* subcomponents go here */}
+</Tabs>;
 
 // OR
 
-// model
-const model = useTabsModel(config)
+// hoisted model
+const model = useTabsModel(config);
 
-<Tabs model={model}>
-  {/* sub components go here */ }
-</Tabs>
+<Tabs model={model}>{/* subcomponents go here */}</Tabs>;
 ```
 
-#### Component Props
+#### Props
 
 <PropsTable of={Tabs} />
 
----
-
 ### Tabs.List
 
-`Tabs.List` is a `div` element with a `role=tablist`. It is a container for `Tabs.Item` components.
-Any additional attributes provided will be forwared to the dom element.
-
 #### Usage
 
+`Tabs.List` is a `div` element with a `role=tablist`. It is a container for `Tabs.Item`
+subcomponents.
+
 ```tsx
 <Tabs.List>{/* Tabs.Items go here */}</Tabs.List>
 ```
 
-#### Component Props
+#### Props
 
-<PropsTable of={Tabs.List} />
+Undocumented props are spread to the underlying `div` element.
 
----
+<PropsTable of={Tabs.List} />
 
 ### Tabs.Item
 
+#### Usage
+
 `Tabs.Item` is a `button` element with a `role=tab`. Each `Tabs.Item` is associated with a
 `Tabs.Panel` either by a `name` attribute or by the position in the list. If a `name` is provided,
-it must match the name passed to the `Tabs.Panel` component.
-
-#### Usage
+it must match the name passed to the corresponding `Tabs.Panel` component.
 
 ```tsx
 <Tabs.Item name="first">First Tab</Tabs.Item>
 ```
 
-#### Component Props
+#### Props
 
-<PropsTable of={Tabs.Item} />
+Undocumented props are spread to the underlying `button` element.
 
----
+<PropsTable of={Tabs.Item} />
 
 ### Tabs.Panel
 
+#### Usage
+
 `Tabs.Panel` is a `div` element with a `role=tabpanel`. It is focusable by default. Each
 `Tabs.Panel` is controlled by a `Tabs.Item` either by a `name` attribute or by the position in the
-list. If a `name` is provided, it must match the name passed to the `Tabs.Item` component.
-
-#### Usage
+list. If a `name` is provided, it must match the name passed to the corresponding `Tabs.Item`
+component.
 
 ```tsx
 <Tabs.Panel name="first">{/* contents of the tab panel */}</Tabs.Panel>
 ```
 
-#### Component Props
+#### Props
+
+Undocumented props are spread to the underlying `div` element.
 
 <PropsTable of={Tabs.Panel} />
 
+## Hooks
+
+### useTabsModel
+
+#### Usage
+
+Creates a `TabsModel` using the provided configuration object.
```

Comment:
Most of the changes in this PR are me shifting around and wordsmithing content, but the `Hooks` section is all new. Let me know if you think I'm missing or misrepresenting anything here.